app/controllers/permalinks_controller.rb

```diff
@@ -17,4 +17,18 @@ def show
     end
   end
 
+  def check
+    permalink = Permalink.find_by_url(params[:path])
```

Comment:
should we handle case where path is not set ? Especially if used through an API, it would raise a 404 which is slightly unexpected. As it's more an invalid param.

Reply:
find_by_* will return nil instead of throwing.

Reply:
yes but the line after your raise no ?

Reply:
Oh, you mean returning something like found: false? I don't think this is too bad of an API as it is right now.

Reply:
I actually think 404 is fine in this case.

Reply:
Moot: we now return json with the 404 body directly on not found.